packages/block-editor/src/components/inserter/search-results.js

```diff
@@ -108,8 +130,9 @@ function InserterSearchResults( {
 		blockTypes,
 		blockTypeCategories,
 		blockTypeCollections,
-		maxBlockTypes,
```

Comment:
I see we had `maxBlockTypes` and `maxBlockTypesToShow` both existed in code. Why did we remove it here?

Reply:
🆗